app/models/miq_set.rb

```diff
@@ -0,0 +1,66 @@
+class MiqSet < ApplicationRecord
+  # self.abstract_class = true
+  self.inheritance_column = :set_type
```

Comment:
if we are modifying the schema for this, do we want to rename this to type while we are at it?

Reply:
:+1: for this.  Though we'd have to change all of the callers.